msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -425,7 +425,46 @@ else { /* handle case where capability isn't supported */ }
 
 ```
 
-#### Generate a deep link to a call
+## Generate a deep link to share content to stage in meetings
+
+You can also generate a deep link to [share the app to stage](~/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md#share-entire-app-to-stage) and start or join a meeting.
+
+When deep link is selected in an app and user is already in a meeting, the app is shared to stage, and a permission pop-up window appears. Users can grant permissions for the participants such as co-editing a document or collaborating with an app.
+
+> [!NOTE]
+> Permission pop-up window appears only for Microsoft 365 apps.
+
+When user isn't in a meeting then user is redirected to the Teams calendar where user needs to join a meeting or an instant meeting (Meetnow) can be initiated, where user can add participants and they can interact with an app.
+
+| Deep link | Format | Example |
+|-----------|--------|---------|
+|To share the app in an ongoing meeting| `msteams:/l/meeting-share?deeplinkId=GUID&fqdn=string&appContext={"appSharingUrl" : "", "appId": ""}` | `https://teams.microsoft.com/l/meeting-share?deeplinkId=ACCC7AFE-449D-4AA3-8D3E-E8A7B3NB1280&fqdn=&lm=deeplink&appContext={ "appSharingUrl" : "https://teams.microsoft.com/extensibility-apps/meetingapis/view", "appId": "9ec80a73-1d41-4bcb-8190-4b9eA9e29fbb"}` |
+|To share the app and open Teams calendar| `msteams:/l/meeting-share?deeplinkId=GUID&fqdn=string&appContext={"appSharingUrl" : "", "appId": "", “useMeetNow”: false }` | `https://teams.microsoft.com/l/meeting-share?deeplinkId=ACCC7AFE-449D-4AA3-8D3E-E8A7B3NB1280&fqdn=&lm=deeplink&appContext={ "appSharingUrl" : "https://teams.microsoft.com/extensibility-apps/meetingapis/view", "appId": "9ec80a73-1d41-4bcb-8190-4b9eA9e29fbb" , “useMeetNow”: false}` |
+|To share the app and initiate instant meeting|`msteams:/l/meeting-share?deeplinkId=GUID&fqdn=string&appContext={"appSharingUrl" : "", "appId": "", “useMeetNow”: true }` | `https://teams.microsoft.com/l/meeting-share?deeplinkId=ACCC7AFE-449D-4AA3-8D3E-E8A7B3NB1280&fqdn=&lm=deeplink&appContext={ "appSharingUrl" : "https://teams.microsoft.com/extensibility-apps/meetingapis/view", "appId": "9ec80a73-1d41-4bcb-8190-4b9eA9e29fbb" , “useMeetNow”: true}` |
+
+The query parameters are:
+
+* `deepLinkId`: GUID or UUID used for telemetry correlation.
+* `fqdn`: Teams supports multi-tenant multi-account, and Teams can run AAD and MSA accounts at the same time. `fqdn`( teams.microsoft.com or teams.live.com ) is needed in Teams deeplink service for tenant or account authentication.
```

Comment:
Please enumerate the supported FQDNs and when they should be used. How do apps know which one to use?

Reply:
@v-ajiths / @v-npaladugu :  Please rephrase this paragraph. Functionality of FQDN: It is needed for tenant or account authentication. Supported values:  refer to the documentation shared. 
![image](https://user-images.githubusercontent.com/31720440/181407006-a5ffeab9-616d-4602-9ffa-23ff4b8f702e.png)

Reply:
Edited and updated

Reply:
Ok, but can we include prescriptive guidance on how apps are supposed to determine which FQDN to use? If an app is constructing this deep link, how do they determine if the meeting was scheduled with TFL, TFW(Public), TFW(GCCH), TFW(DOD)?

Reply:
Updated the content

Reply:
Still this is unclear in terms of app guidance- How does the app know if the meeting was scheduled for TFL, TFW commercial, or TFW gov?